### ISSUE TITLE
Edit profile screen

### ISSUE THREAD

**Issue by jiakeliu** [gunnertech/thumbwar-ios-client#87]
User's edit profile screen. Mock-up to follow.

**Comment by jiakeliu** [comment]
Required fields:

Photo, Name, Username, Mobile Number, Email, Password.

**Comment by jiakeliu** [comment]
Design mockup:

![editprofile](https://cloud.githubusercontent.com/assets/1039583/7243365/22020910-e781-11e4-9c61-7f6ad66b6e5b.jpg)

![changepassword](https://cloud.githubusercontent.com/assets/1039583/7243368/27532af2-e781-11e4-9ddc-b16bde8e6717.jpg)

**Comment by jiakeliu** [comment]
PSD files: https://www.dropbox.com/sh/vme0xozktp3yils/AACTllazlNjOpawXdI6T6uwSa?dl=0

**Comment by husseinm** [comment]
We still want this? I have never seen it and it is not at all implemented in the code and would have to be a new addition, at lease that I am aware of.

**Comment by jiakeliu** [comment]
Yes, we need to implement this after the design match up. We will talk about this in the sprint planning.

**Comment by jiakeliu** [comment]
@husseinm Can we fit this ticket in this iteration?

**Comment by husseinm** [comment]
Probably not

**Comment by jiakeliu** [comment]
Moved to Backlog for now.

**Comment by jiakeliu** [comment]
Discussed and agreed to add a Settings (gear) icon on user profile screen, which leads to a new screen with Logout as the only option.
